docs/src/Box.doc.js

```diff
@@ -357,18 +372,20 @@ const MarginSwatch = (props: *) => (
 );
 
 card(
-  <Box
+  <Card
     description={`
     Margins are applied in boints and are asymmetric. Margins can be -6 to 6 boints. You can set margin to be auto in the left and right axis.
 
     In the example on the right the outer \`Box\` is in transparent red, and the inner \`Box\` is transparent blue.
 
+    Instead of using \`marginLeft\` and \`marginRight\`, opt for the RTL language aware \`marginStart\` and \`marginEnd\`.
```

Comment:
its not very clear from this example what `marginStart` and `marginEnd` gives us.  i would suggest adding a separate `RTL` section and showing an example of how marginStart and marginEnd work in relation with html dir